File: Area Amministratore/GiocoDislessia/src/model/Distrattore.java
package model;

public class Distrattore {
	private int id;
	private String stringa;
	
	public Distrattore(String stringa)
	{
		this.stringa=stringa;
	}
	
	public Distrattore(int id, String stringa)
	{
		this.id=id;
		this.stringa=stringa;
	}
	
	public int getId() {
		return id;
	}
	
	public void setId(int id) {
		this.id = id;
	}
	
	public String getStringa() {
		return stringa;
	}
	
	public void setStringa(String stringa) {
		this.stringa = stringa;
	}


}

File: Gioco in Unity/GiocoDislessiaRemotoServlet/Assets/Bean/Distrattore.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class Distrattore
{
    private int id;
    private string parola;

    public Distrattore(int id, string parola)
    {
        this.id = id;
        this.parola = parola;
    }

    public int getId()
    {
        return id;
    }

    public void setId(int id)
    {
        this.id = id;
    }

    public string getParola()
    {
        return parola;
    }

    public void setParola(string parola)
    {
        this.parola = parola;
    }

    public bool cercaDistrattore(string distrattore)
    {
        if(this.getParola()==distrattore)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}
